Neos.Flow/Classes/Persistence/Doctrine/Mapping/Driver/FlowAnnotationDriver.php

```diff
@@ -775,7 +776,7 @@ protected function evaluatePropertyAnnotations(ORM\ClassMetadataInfo $metadata)
                         'allocationSize' => $seqGeneratorAnnotation->allocationSize,
                         'initialValue' => $seqGeneratorAnnotation->initialValue
                     ]);
-                } elseif ($this->reader->getPropertyAnnotation($property, ORM\TableGenerator::class) !== null) {
```

Comment:
This is "correct", carried over from https://github.com/doctrine/doctrine2/blob/master/lib/Doctrine/ORM/Mapping/Driver/AnnotationDriver.php#L332-L333 - it could be dropped, but then again, our driver is deliberately kept as close to the original one to ease updates…